### PR TITLE
fifth commit

### PR DESCRIPTION
**Pull request by AbhiTagline**
Question 5

### CODE REVIEW

File: question5.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+numbers = [9,4,8,10,2,4,8,3,14,4,8]
+
+num = 12
+
+pairs = []
+
+for i in range(len(numbers)):
+    for j in range(len(numbers)):
```
Comment (himanshu-suthar-tagline) [inline question5.py:8]: Can't I do without using index? Like accessing elements directly from numbers list in loop?

File: question5.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+        if numbers[i]+numbers[j]== num and sorted([numbers[i],numbers[j]]) not in pairs:
```
Comment (himanshu-suthar-tagline) [inline question5.py]: Make sure you have code formatted properly. Before and after `==`, there must be one space.

It is good to put `sorted([numbers[i],numbers[j]])` in a variable, as you are using it twice.

File: question5.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+            pairs.append(sorted([numbers[i],numbers[j]]))
+print((pairs))
```
Comment (himanshu-suthar-tagline) [inline question5.py]: Why you need to put () around pairs?

**Comment by AbhiTagline** [comment]
> Question 5

Done with the changes please recheck and confirm

**Comment by AbhiTagline** [comment]
> > Question 5

**Pull request closed without merging**